README.md

```diff
@@ -60,6 +60,14 @@ TBD
 
 # Changelist
 
+* Version update to 2.4.1-2
+  
+  * Add local control annotations via deepAnnotation pipe
```

Comment:
If you annotate each change as "Major", "Minor", or "Patch" the future task of understanding whether a changed was breaking backwards compatibility is much simplified. Now your knowledge of the change is fresh.

This is the addition of columns to the end of a table. So it's a minor change.

The column swap bug fix before should actually be a major change (even though we did not increase the version number accordingly, which has other reasons.

For other workflows I actually went through all the changes and post-annotated each changelog entry as major, minor, or patch.